src/build/__init__.py

```diff
@@ -133,10 +133,11 @@ def __init__(self, srcdir='.', config_settings=None):  # type: (str, Optional[Co
 
         self._backend = self._build_system['build-backend']
 
-        try:
-            importlib.import_module(self._backend.split(':')[0])
-        except ImportError:  # can't mock importlib.import_module  # pragma: no cover
-            raise BuildException("Backend '{}' is not available".format(self._backend))
+        if 'backend-path' not in self._build_system:
+            try:
+                importlib.import_module(self._backend.split(':')[0])
+            except ImportError:  # can't mock importlib.import_module  # pragma: no cover
```

Comment:
Why can't we mock it?

Reply:
I couldn't get it to work. Maybe it was just be not doing something wrong, I remember that I spent a fair bit of time here, but couldn't make it work.

I gave it another spin today and it was actually pretty straight forward.